components/infra-proxy-service/integration_test/environments_test.go

```diff
@@ -101,6 +104,188 @@ func TestGetEnvironments(t *testing.T) {
 
 }
 
+func TestGetEnvironment(t *testing.T) {
+	// rpc GetEnvironment (request.Environment) returns (response.Environment)
+	ctx := context.Background()
+	t.Run("when the environment exists, return the environment successfully", func(t *testing.T) {
+		name := fmt.Sprintf("chef-environment-%d", time.Now().Nanosecond())
+		creReq := &request.CreateEnvironment{
+			ServerId:    autoDeployedChefServerID,
+			OrgId:       autoDeployedChefOrganizationID,
+			Name:        name,
+			Description: "auto generate chef environment",
+			CookbookVersions: map[string]string{
+				"audit":       ">= 9.5.0",
+				"chef-client": "= 12.3.3",
+			},
+			DefaultAttributes: &structpb.Struct{
+				Fields: map[string]*structpb.Value{
+					"attribute1": &structpb.Value{Kind: &structpb.Value_StringValue{StringValue: "value"}},
+				},
+			},
+		}
+		envRes, err := infraProxy.CreateEnvironment(ctx, creReq)
+		assert.NoError(t, err)
+		assert.NotNil(t, envRes)
+
+		req := &request.Environment{
+			ServerId: autoDeployedChefServerID,
+			OrgId:    autoDeployedChefOrganizationID,
+			Name:     name,
+		}
+		res, err := infraProxy.GetEnvironment(ctx, req)
+		assert.NoError(t, err)
+		assert.NotNil(t, res)
+		assert.Equal(t, name, res.GetName())
+		assert.Equal(t, "auto generate chef environment", res.GetDescription())
+		assert.Equal(t, creReq.CookbookVersions, res.CookbookVersions)
+	})
+}
+
+func TestCreateEnvironment(t *testing.T) {
```

Comment:
Please add a test to cover Ovverride attributes too. I can only see defaultAttributes

Reply:
Added asset for both default & override attributes. Thanks!

```
assert.Equal(t, "{\"attribute1\":\"value\"}", res.DefaultAttributes)
assert.Equal(t, "{\"attribute1\":\"override\"}", res.OverrideAttributes)
```